s3/src/main/mima-filters/2.0.0-M3.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# MiMa upgrade
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.alpakka.s3.S3Settings.this")
```

Comment:
So this is a binary incompatibility that wasn't detected on 0.6.4, but which is OK since we're starting a new major version anyway? Might be better to explain the change in the comment rather than just 'mima upgrade', but doesn't matter much since we're not promising binary compatibility here anyway.

Reply:
You're right. (This one is even `private[alpakka]`, so it is not an API change anyway.)